sale_report_delivered_subtotal/reports/sale_report.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleReport(models.Model):
+    _inherit = "sale.report"
+
+    price_subtotal_delivered = fields.Float(
+        string='Total Price Delivered',
+    )
+
+    def _query(self, with_clause='', fields={}, groupby='', from_clause=''):
+        fields = dict(fields or {})
```

Comment:
Adapted to the above code:
```suggestion
        if fields is None:
            fields = {}
```

Reply:
Actually it would work as it is.

Reply:
It's for not creating a new one if not needed and then build a new dict for it, but it's not critical indeed.